test/sequential/xyk-pallet.api.maintenance-config.test.ts

```diff
@@ -264,3 +274,75 @@ async function getSudoError(
 
   expect(sudoEventError.name).toEqual(expectedError);
 }
+
+test("maintenance- bootstrap can be run in maintenanceMode, but you can't provide it and the pool will not be created", async () => {
+  await checkMaintenanceStatus(false, false);
+  await checkLastBootstrapFinalized(sudo);
+  const bootstrapCurrency = await createNewBootstrapCurrency(sudo);
+  const waitingPeriod = 10;
+  const bootstrapPeriod = 8;
+
+  [testUser1] = setupUsers();
+
+  await setupBootstrapTokensBalance(bootstrapCurrency, sudo, [testUser1]);
+
+  await sudo.mint(bootstrapCurrency, testUser1, toBN("1", 20));
+
+  await Sudo.batchAsSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      foundationAccountAddress,
+      Maintenance.switchMaintenanceModeOn(),
+    ),
+  );
+
+  await checkMaintenanceStatus(true, false);
+
+  await scheduleBootstrap(
+    sudo,
+    GASP_ASSET_ID,
+    bootstrapCurrency,
+    waitingPeriod,
+    bootstrapPeriod,
+  ).then((result) => {
+    const eventResponse = getEventResultFromMangataTx(result);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  });
+
+  await waitForBootstrapStatus("Public", waitingPeriod);
+
+  // check that user can make provision while bootstrap running
+  await provisionBootstrap(
+    testUser1,
+    bootstrapCurrency,
+    BN_HUNDRED_BILLIONS,
+  ).then((result) => {
+    const eventResponse = getEventResultFromMangataTx(result);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicFailed);
+    expect(eventResponse.data).toEqual("ProvisioningBlockedByMaintenanceMode");
+  });
+
+  await waitForBootstrapStatus("Finished", bootstrapPeriod);
+  await claimRewardsBootstrap(testUser1).then((result) => {
+    const eventResponse = getEventResultFromMangataTx(result);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicFailed);
+    expect(eventResponse.data).toEqual(xykErrors.MathOverflow);
+  });
+
+  const foundationSwitchModeEvents = await Sudo.batchAsSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      foundationAccountAddress,
+      Maintenance.switchMaintenanceModeOff(),
+    ),
+  );
+
+  const filteredEvent = foundationSwitchModeEvents.filter(
+    (extrinsicResult) => extrinsicResult.method === "SudoAsDone",
+  );
+  const eventIndex = JSON.parse(JSON.stringify(filteredEvent[0].event.data[0]));
+  expect(eventIndex.ok).toBeDefined();
+
+  const poolId = await getLiquidityAssetId(GASP_ASSET_ID, bootstrapCurrency);
+  expect(poolId).bnEqual(new BN(-1));
```

Comment:
this can be a side effect of no one being able to provide liq. Can you do another test that starts maintenance after a a provision being done? ie: 
1,- CheckMaintenanceOff
2,- Start bootstrap 
3,- Wait for Public phase and provision with one user
4,- start maintenance 
5,- wait for boostrap finished. 
6,- Check that bootstrap did not / did create the pool.

Reply:
The scenario was created, but we have a bug here https://mangatafinance.atlassian.net/browse/GASP-2210